app/(home)/page.js

```diff
@@ -0,0 +1,26 @@
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import React from "react";
+import { FaAngleRight, FaUserAlt } from "react-icons/fa";
+
+export default function page() {
+  return (
+    <div className="text-center relative max-w-2xl mx-auto">
+      <h1 className="text-light-1 text-heading1-bold">
+        Discover Your Next Favorite Movie with a Friend
+      </h1>
+      <h4 class="sub-title text-white text-[20px] md:text-[24px] leading-[1.13] mt-[27px] mb-[28px] font-normal">
+        Enjoy anywhere. savor entertainment together
```

Comment:
Small thing but I'd change to a capital S - we can do a read through of all the text later though and pick up any spelling/grammar things :)